clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -23008,6 +23050,7 @@ llvm::CallInst *CodeGenFunction::EmitFPBuiltinIndirectCall(
       FPAccuracyIntrinsicID = Intrinsic::fpbuiltin_exp2;
       break;
     case Builtin::BIexpm1:
+    case Builtin::BIexpm1f:
```

Comment:
For my own curiosity, how did you (or how can one) convince yourself that the entries are complete?   Is there a list of functions you were working from?

Reply:
There is no list AFAIK, and there are probably still some functions (especially the single precision ones) that are still missing.